python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -329,6 +340,24 @@ def config_type(cls):
                     " always be set by the run launcher."
                 ),
             ),
+            "retries": Field(
```

Comment:
the name here might cause confusion with the existing deployment-level run retries feature: https://docs.dagster.io/deployment/run-retries#run-retries

Maybe some name to indicate that it specifically for retries at the AWS level when launching the run

like "ecs_retries" or "launch_retries" or something

Reply:
I think for something relatively low-level like this you could also make it an env var vs. instance-level config